### ISSUE TITLE
cerebellin image broken due to dependency on isa-rwval on develop!

### ISSUE THREAD

**Issue by pcm32** [phnmnl/container-galaxy-k8s-runtime#186]
Unfortunately we didn't noticed that isa-rwval was pinned against its develop branch on requirements.txt:

```
# GenomeSpace dependencies
python-genomespaceclient==0.1.8
pykube==0.15.0
-e git://github.com/ISA-tools/isa-rwval.git@develop#egg=isatools
```

We should probably hotfix that release image (and bless the new resulting image) to a defined version of isa-rwval, ideally where dependencies networkx and six versions are fixed.

**Comment by djcomlab** [comment]
This should be fixed in PR https://github.com/phnmnl/container-galaxy-k8s-runtime/pull/176

**Comment by djcomlab** [comment]
Fixed in cb86b61
